### PR TITLE
Tim best/fix test db

### LINKED ISSUE #661: Admins should go directly to admin console
As a site admin user with no organization access, I should be sent to the admin console immediately upon logging in. 

Depends on #654.

### PR DESCRIPTION
**Pull request by timothybest-usds**
## Related Issue or Background Info

Changes that came about from fixing the api in the test environment. The API schemas and tables got clobbered when testing out the MetaBase integration. 

UI changes are so I can setup organizations in `test` via the admin port instead of the DB

## Changes Proposed

- Updated the terraform setups instructions in the ops README
- Added steps I took to take down and rebuild the database in the test env
- Allow the admin portal to be accessed when a user is not part of an org
- Redirects site admins to the site admin page (resolves #661)
- Removes check for facility name uniqueness on first facility

### CODE REVIEW

**Review by jeskew-gov — APPROVED** [review]

**Comment by benwarfield-usds** [comment]
So if I'm reading this correctly, this allows the UI to function for admin users without the admin users being part of an org? I think that's like one line of JavaScript away from resolving #661 and doing away with the need for a "Prod Admins" organization. 🙂

File: backend/src/main/java/gov/cdc/usds/simplereport/api/organization/OrganizationMutationResolver.java [hunk 1]
```diff
@@ -131,7 +131,6 @@ public Organization createOrganization(String name, String externalId, String te
             String orderingProviderCity, String orderingProviderCounty, String orderingProviderState,
             String orderingProviderZipCode, String orderingProviderTelephone, List<String> deviceIds,
             String defaultDeviceId) {
-        _os.assertFacilityNameAvailable(testingFacilityName);
```
Comment (timothybest-usds) [inline backend/src/main/java/gov/cdc/usds/simplereport/api/organization/OrganizationMutationResolver.java:134]: This get the users current organization. Making it impossible to add an organization when you arn't in an organization

File: frontend/src/app/App.tsx [hunk 2]
```diff
@@ -114,7 +114,12 @@ const App = () => {
                   path="/"
                   exact
                   render={({ location }) => (
-                    <Redirect to={{ ...location, pathname: "/queue" }} />
+                    <Redirect
+                      to={{
+                        ...location,
+                        pathname: data.whoami.isAdmin ? "/admin" : "/queue",
```
Comment (benwarfield-usds) [inline frontend/src/app/App.tsx:120]: 🎉

File: backend/src/main/java/gov/cdc/usds/simplereport/api/organization/OrganizationMutationResolver.java (hunk 1 above)
Comment (benwarfield-usds) [reply backend/src/main/java/gov/cdc/usds/simplereport/api/organization/OrganizationMutationResolver.java:134]: I would advocate removing the method entirely, and doing a better job of wrapping the error we get from the unique key constraint violation, but that's not a today solution. 😄

**Review by benwarfield-usds — APPROVED** [review]
LGTM, albeit I'm just sort of assuming that the typescript changes do what they look like they do. 😁

**Pull request merged**